lib/jdomodels/src/main/java/org/sagebionetworks/repo/model/dbo/schema/DBOJsonSchemaBindObject.java

```diff
@@ -42,6 +44,7 @@
 	private String objectType;
 	private Long createdBy;
 	private Timestamp createdOn;
+	private Boolean enableDerived;
```

Comment:
We need a piece of migration code to explicitly check if this value is null, in which case we set it as false (otherwise we end up with null values in the DB).